spec/models/author_spec.rb

```diff
@@ -129,4 +129,53 @@
     author.valid?
     expect(author.errors[:name]).to eq([])
   end
+
+  describe ".zodiac" do
+    it "returns the zodiac that corresponds to the birthday" do
+
+      {
+        "Aries ♈️" => ["March 21", "April 19"],
+        "Taurus ♉️" => ["April 20", "May 20"],
+        "Gemini ♊️" => ["May 21", "June 20"],
+        "Cancer ♋️" => ["June 21", "July 22"],
+        "Leo ♌️" => ["July 23", "August 22"],
+        "Virgo ♍️" => ["August 23", "September 22"],
+        "Libra ♎️" => ["September 23", "October 22"],
+        "Scorpio ♏️" => ["October 23", "November 21"],
+        "Sagittarius ♐️" => ["November 22", "December 21"],
+        # "Capricorn ♑️" => ["December 22", "January 19"],
+        "Aquarius ♒️" => ["January 20", "February 18"],
+        "Pisces ♓️" => ["February 19", "March 20"],
+      }.each { |zodiac, (begin_date, end_date)|
+        author = create(:author, date_of_birth: Date.parse("#{ begin_date } 2002"))
+        expect(author.zodiac).to eq(zodiac)
+        author.update!(date_of_birth: Date.parse("#{ end_date } 2002"))
+        expect(author.zodiac).to eq(zodiac)
+      }
+
+      # Because Capricorn spans 2 years, a comparison is more complex than the other zodiacs
+      # To ensure it is properly tested, we are testing explicitly below
+
+      ["2002-01-01", "2002-01-19", "2002-12-22", "2002-12-31"].each { |date|
+        capricorn_author = create(:author, date_of_birth: date)
+        expect(capricorn_author.zodiac).to eq("Capricorn ♑️")
+      }
+    end
+  end
+
+  describe ".age_at_first_release" do
+    it "Calculates age at first release" do
+      author = create(:author, date_of_birth: "Sun, 15 Dec 1980")
```

Comment:
I'd try and keep the data format you use consistent, that way it's easy to know which format you should use in future, and it's easier for people reading the code.